libs/dh/globalization/configuration-localization/src/lib/dh-transloco.module.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ModuleWithProviders,
+  NgModule,
+  Optional,
+  SkipSelf,
+} from '@angular/core';
+import { dhTranslocoHttpLoaderProvider } from '@energinet-datahub/dh/globalization/data-access-localization';
+import { environment } from '@energinet-datahub/dh/shared/environments';
+import {
+  TRANSLOCO_CONFIG,
+  translocoConfig,
+  TranslocoModule,
+} from '@ngneat/transloco';
+
+@NgModule({
+  imports: [TranslocoModule],
+  providers: [
+    {
+      provide: TRANSLOCO_CONFIG,
+      useValue: translocoConfig({
+        availableLangs: ['da', 'en'],
+        defaultLang: 'da',
+        fallbackLang: ['da', 'en'],
+        flatten: {
+          aot: environment.production,
+        },
+        missingHandler: {
+          useFallbackTranslation: true,
+        },
+        // Remove this option if your application doesn't support changing language in runtime.
+        reRenderOnLangChange: true,
```

Comment:
question: what happens if this option is removed? Will Transloco refresh the page in order to load the new translations?

Reply:
I'm not sure but I think we would have to refresh the page ourselves as it removes some change detection integrations as far as I understand.

>It can save on memory by rendering the view once and unsubscribing from the language changes event (defaults to false):
[From Transloco documentation](https://ngneat.github.io/transloco/docs/config-options/#rerenderonlangchange)